backend/src/main/java/hu/flowacademy/worksheet/service/UserService.java

```diff
@@ -49,4 +52,13 @@ private void validateUser(User user) throws ValidationException {
     public List<User> findUserByNameAndEmail(String searchPart) {
         return userRepository.findByEmailContainingOrFirstNameContainingOrLastNameContaining(searchPart, searchPart, searchPart);
     }
+
+    public List<User> listRegistrations(Pageable pageable) {
```

Comment:
Vond össze a `findUserByNameAndEmail`, `listRegistrations`, `listRegistrations` fügvényeket egy `list(...)` függvénnyé, kapja meg az összes szükséges paramétert.

Reply:
Itt csináltam az én részemből 1 egészet. Patrickét nem tudom h hogy néz ki pontosan. Igy megint nem nyúltam hozzá. Elég az enyém, vagy várjam ba a @nomansnick verziót is